src/test/kotlin/pl/droidsonroids/gradle/jacoco/testkit/JaCoCoTestKitPluginFunctionalTest.kt

```diff
@@ -20,6 +19,10 @@ class JaCoCoTestKitPluginFunctionalTest {
         temporaryFolder.newFile("gradle.properties").fillFromResource("testkit-gradle.properties")
     }
 
+    companion object {
+        const val generateJacocoTestKitProperties = "generateJacocoTestKitProperties"
```

Comment:
Is it used anywhere?
I see only the same strings below. Not references.

Reply:
It's used, I forgot to use it in my new tests, but now all the tests use it (the words is exactly the same, so only the quotes change)